service/controller/v10/credential/credential.go

```diff
@@ -0,0 +1,75 @@
+package credential
+
+import (
+	"encoding/json"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/api/core/v1"
+	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/aws-operator/service/controller/v10/key"
+)
+
+const (
+	rolesKey    = "roles"
+	rolePurpose = "aws-operator"
+)
```

Comment:
Probably subject to change after newly agreed secret format.

Reply:
done in https://github.com/giantswarm/aws-operator/pull/933/commits/52a462e67364282a325114bf0532e4c8a9412240